### ISSUE TITLE
Avoid using Jeet in favor for flexbox and/or css-grid

### ISSUE THREAD

**Issue by bdbch** [flyntwp/flynt-starter-theme#292]
Currently Flynt still uses jeet as a layouting solution which I think is obsolete since last year. Most browsers already support flexbox and even IE10 and IE11 can run flexbox quite well if the autoprefixer is doing his job right.

The current global browser support is at 95.8%  and unprefixed browser support is at 94.75%
See here: https://caniuse.com/#feat=flexbox

The CSS-grid support looks almost the same except some older versions of chrome and firefox still don't support css-grid. Still css-grid is already at 87.85% support:
https://caniuse.com/#search=css-grid

I would assume flexbox is already safe to use for almost every modern project and css-grid will become the norm in 1-2 years too which will make any layout library obsolete.

Looking forward to hear from the team and your opinions.

**Comment by Qakulukiam** [comment]
I agree and we already mostly use flexbox internally. I will talk to the team about removing the jeet dependency with the next update and keep you posted.
